Intersect.Server/Database/Item.cs

```diff
@@ -74,6 +55,7 @@ public Item(Item item) : this(item.ItemId, item.Quantity, item.BagId, item.Bag)
 
         public int Quantity { get; set; }
 
+        //DEPRECATED
```

Comment:
Actual deprecation should be via `[Obsolete]` (and it would need to be on StatBuffs not StatBuffsJson), but if it's being replaced it should just be removed, even if that is a pain in the ass.

Even if it's only being deprecated though, you aren't copying the old data into the new ItemProperties, and so this change is effectively causing data loss.

Can you please check and see what ItemProperties looks like when serialized as JSON and stored in the DB when it does have ItemProperties set? This will help write the SQL-only migration to ItemProperties.

Reply:
![image](https://user-images.githubusercontent.com/63019821/198119466-25f4be2b-d82f-4798-a56a-e5fa532c2e6b.png)

Reply:
And an item that uses the old and the new
![image](https://user-images.githubusercontent.com/63019821/198119628-5e9b8277-58cd-4228-8bab-f28f334fa41f.png)